packages/SwingSet/test/test-transcript-light.js

```diff
@@ -3,97 +3,105 @@
 import { test } from '../tools/prepare-test-env-ava';
 
 import path from 'path';
-import {
-  initSwingStore,
-  getAllState,
-  setAllState,
-} from '@agoric/swing-store-simple';
+import { getAllState, setAllState } from '@agoric/swing-store-simple';
+import { provideHostStorage } from '../src/hostStorage';
 import { buildVatController, loadBasedir } from '../src/index';
 
 test('transcript-light load', async t => {
   const config = await loadBasedir(
     path.resolve(__dirname, 'basedir-transcript'),
   );
-  const { storage } = initSwingStore();
-  const c = await buildVatController(config, ['one'], { hostStorage: storage });
-  const state0 = getAllState(storage);
+  const hostStorage = provideHostStorage();
+  const c = await buildVatController(config, ['one'], { hostStorage });
+  const kvStore = hostStorage.kvStore;
+  const state0 = getAllState(kvStore);
   t.is(state0.initialized, 'true');
   t.not(state0.runQueue, '[]');
 
   await c.step();
-  const state1 = getAllState(storage);
+  const state1 = getAllState(kvStore);
 
   await c.step();
-  const state2 = getAllState(storage);
+  const state2 = getAllState(kvStore);
 
   await c.step();
-  const state3 = getAllState(storage);
+  const state3 = getAllState(kvStore);
 
   await c.step();
-  const state4 = getAllState(storage);
+  const state4 = getAllState(kvStore);
 
   await c.step();
-  const state5 = getAllState(storage);
+  const state5 = getAllState(kvStore);
 
   // build from loaded state
   // Step 0
 
   const cfg0 = await loadBasedir(path.resolve(__dirname, 'basedir-transcript'));
-  const storage0 = initSwingStore().storage;
-  setAllState(storage0, state0);
-  const c0 = await buildVatController(cfg0, ['one'], { hostStorage: storage0 });
+  const hostStorage0 = provideHostStorage();
+  const kvStore0 = hostStorage0.kvStore;
+  // XXX TODO copy transcripts
```

Comment:
yeah, I'm thinking the simpleStore wants a debug/test method named `copy` that gets you everything